### ISSUE TITLE
Get rid of base/Makefile

### ISSUE THREAD

**Issue by sroettger** [google/kctf#176]
The reason to have a Makefile to start challenges was that it can resolve the dependencies for us and perform build actions in parallel and only build what's needed.
However, we simplified a bunch of stuff in the Makefile when we moved to operators. And we always need to run docker build since we leave it to the docker daemon to decide if it needs to rebuild any of the layers.

That only leaves the advantage that healthcheck and challenge image can be built in parallel easily with the Makefile.

The downside however is that Makefiles are not nice to work with and it's pretty hard to follow the flow, make sure that all dependencies are correct etc.
A few bash scripts would do these things much nicer and we would have one fewer technology that people need to understand.

**Comment by sirdarckcat** [comment]
this is done, right?

@sroettger
